src/main/java/com/bfrisco/benchants/utils/ItemInfo.java

```diff
@@ -0,0 +1,86 @@
+package com.bfrisco.benchants.utils;
+
+import org.bukkit.inventory.ItemStack;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ItemInfo {
+
+    public static final String ANCIENT_POWER_ACTIVE = "§8Ancient Power §x§F§F§0§0§4§C♆";
+    public static final String ANCIENT_POWER_INACTIVE = "§8Ancient Power ♆";
+    public static final String ANCIENT_BLUE = "§8Ancient Power§x§6§D§5§E§F§F ♆";
+    public static final String ANCIENT_RED = "§8Ancient Power§x§F§F§0§0§0§0 ♆";
+    public static final String ANCIENT_YELLOW = "§8Ancient Power§x§F§F§E§C§2§7 ♆";
+
+    public static final List<String> IMBUED_LORE = new ArrayList<>(){
+        {
+        add(ANCIENT_POWER_ACTIVE);
+        add(ANCIENT_POWER_INACTIVE);
+        }
+    };
+    public static final List<String> UNIMBUED_LORE = new ArrayList<>(){
+        {
+        add(ANCIENT_RED);
+        add(ANCIENT_YELLOW);
+        add(ANCIENT_BLUE);
+        }
+    };
+
+    public static boolean isTitanTool(ItemStack item){
+
+        List<String> loreList = item.getItemMeta().getLore();
+
+        if (loreList == null || !item.hasItemMeta()) return false;
+        for (String lore : loreList) {
+            //detects for any variant of ancient power color in titan tools
+            //then returns true if any exist
+            if (ItemInfo.UNIMBUED_LORE.contains(lore)) {

```

Comment:
We can simplify this to:
```
if (UNIMBUED_LORE.contains(lore) || IMBUED_LORE.contains(lore)) return true;
```

Since this method is in the `ItemInfo` class, no need to reference it